src/api/app/views/webui/users/tokens/show.html.haml

```diff
@@ -0,0 +1,64 @@
+- @pagetitle = 'Show Token'
+
+.card
+  .card-body
+    %h3.mb-3= @pagetitle
+
+    .row
+      .col-12.col-md-10.col-lg-8
+        %p.font-weight-bold Your new OBS Personal Access Token
+
+        .form-row
+          .col-sm
+            %label.font-weight-bold
+              Id:
+            %span
+              = @token.id
+
+        - if session[:show_token]
+          - session[:show_token] = nil # Ensure we're only showing the token once
+          .form-row
+            .col-12.col-md-10.col-lg-9
+              %fieldset.form-group
+                %label.col-form-label.font-weight-bold
+                  Secret:
+                .input-group
+                  = render CopyToClipboardInputComponent.new(token_string: @token.string)
+              %hr
+
+        .form-row
+          .col
+            .form-group
+              %label.col-form-label.mr-2.font-weight-bold
+                Type:
+              %span
+                = @token.type
+
+        .form-row
+          .col-sm
+            .form-group.ui-front
+              %label.font-weight-bold
+                Name:
+              %span
+                = @token.name
+
+        - if @token.package
+          .form-row
+            .col-sm
+              .form-group.ui-front
+                %label.font-weight-bold
+                  Project name:
+                %span
+                  = @token.package.project.name
+
+          .form-row
+            .col-sm
+              .form-group.ui-front
+                %label.font-weight-bold
+                  Package name:
+                %span
+                  = @token.package.name
+
+        .actions
+          = render partial: 'actions'
+          = link_to('Edit', edit_token_path(@token), title: 'Edit', class: 'btn btn-outline-primary px-4 mr-3 mt-3 mt-sm-0')
```

Comment:
Edit links/buttons should have the `fas.fa-edit` icon as described in the [icons](https://obs-patterns.netlify.app/atoms/icons.html) page of our pattern library.

We tend to have _Edit_ links next to the name/title of object. In this case, it would be right next to the token name (`<h3>` at the top of this view).